tst/com/amazon/corretto/crypto/provider/test/EdDSATest.java

```diff
@@ -55,7 +56,14 @@ public void uniqueKeyGen() {
     final KeyPair kp1 = nativeGen.generateKeyPair();
     final KeyPair kp2 = nativeGen.generateKeyPair();
 
-    assertTrue(!kp1.equals(kp2));
+    final byte [] pk1 = kp1.getPrivate().getEncoded();
```

Comment:
I'm 100% sure the style checker is going to ding you on this. should be `final byte[] foo`